pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml

```diff
@@ -1,18 +1,18 @@
 apiVersion: v1
 kind: Secret
 metadata:
-  name: EXAMPLE_SECRET_NAME
+  name: edge-user-1
 type: Opaque
 stringData:
   # This "token" field is used for interacting with the GitHub API to
   # create a pull request. Change the value to a token from your
   # GitHub-compatible forge.
-  token: EXAMPLE_API_TOKEN
+  token: github_pat_1234567890ABCDAPI_TOKEN
```

Comment:
Since the previous config values (username, gitServer, ...) where using actual values that worked in our demo cluster, should we update the new values that we expect the user to change to be something that will break the manifest deployment if not updated for the custom environment?

Reply:
I'm not sure I follow.

The `EXAMPLE_API_TOKEN` hopefully was not a valid token, nor is `github_pat_1234567890ABCDAPI_TOKEN`. At the same time, with that `github_pat_*` format I try to show the user what they should be after in their GitHub UI.

Reply:
Just to clarify.

The references to a replacement variable like `github_pat_1234567890ABCDAPI_TOKEN` would be changed to  `<GITHUB_PAT_1234567890ABCDAPI_TOKEN>`.  This would make it obvious to the the user that it needs to be updated AND if unchanged it would clearly break in a way to makes it obvious that this value was still applied to the cluster

Reply:
I still don't understand what you suggest. Adding the `<` and `>` there seems to be valid YAML, so `oc apply` still passes:
```bash
$ git diff
diff --git a/pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml b/pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml
index b12f7b5..d14b8dc 100644
--- a/pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml
+++ b/pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml
@@ -7,7 +7,7 @@ stringData:
   # This "token" field is used for interacting with the GitHub API to
   # create a pull request. Change the value to a token from your
   # GitHub-compatible forge.
-  token: github_pat_1234567890ABCDAPI_TOKEN
+  token: <github_pat_1234567890ABCDAPI_TOKEN>
   # This .git-credentials field is used to specify credentials when
   # interacting with a Git server (clone/fetch/push). It will be
   # placed as a file on disk so that the Git CLI can use it, so change
$ oc apply -f pipelines/tekton/gitops-update-pipeline/example-pipelineruns/example-git-credentials-secret.yaml
secret/edge-user-1 created
```

I mean, I can add those there but it does not seem it would achieve the clear break that you are after. :-)

Reply:
Apologies. I had to remember what characters would  break an unquoted yaml string.  It should be `token: {{github_pat_1234567890ABCDAPI_TOKEN}} .  `kustomize` should throw an error

```
Error: map[string]interface {}(nil): yaml: invalid map key: map[string]interface {}{"github_pat_1234567890ABCDAPI_TOKEN":""}
```

Reply:
That works. PR updated to 1a573158c97a7809a27b7be9a6a47586fb43b771.